src/Client/Connection.h

```diff
@@ -66,6 +66,7 @@ struct Packet
     Progress progress;
     BlockStreamProfileInfo profile_info;
     std::vector<UUID> part_uuids;
+    String next_task;
```

Comment:
It is not clear what this string is used for.
Maybe `read_task_id`? Also a comment will be helpful.

Our `Packet` is a messy in general. It is not clear which fields are used for which packets.

Reply:
Fixed